### PR TITLE
[ARM] Add examples for az tag -h

### LINKED ISSUE #10867: Aladdin: Add examples to group -h
For more context, contact JasonSha

Current Experience:
```
$ az tag -h

Group
    az tag : Manage resource tags.

Commands:
    add-value    : Creates a tag value.
    create       : Creates a tag in the subscription.
    delete       : Deletes a tag from the subscription.
    list         : Gets the names and values of all resource tags that are defined in a
                   subscription.
    remove-value : Deletes a tag value.
$
```

Proposed Experience:
```

$ az tag -h

Group
    az tag : Manage resource tags.

Commands:
    add-value    : Creates a tag value.
    create       : Creates a tag in the subscription.
    delete       : Deletes a tag from the subscription.
    list         : Gets the names and values of all resource tags that are defined in a
                   subscription.
    remove-value : Deletes a tag value.

Here are the most common ways to use [az tag]:

Creates a tag in the subscription. (autogenerated)
az tag create --name MyTag

Deletes a tag from the subscription. (autogenerated)
az tag delete --name MyTag

Creates a tag value. (autogenerated)
az tag add-value --name MyTag --value {value}

```

### PR DESCRIPTION
**Pull request by Juliehzl**
Fix #10867
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by qwordy** [comment]
1. Why autogenerated?
2. Can we create a tag and add a value in one command?

File: src/azure-cli/azure/cli/command_modules/resource/_help.py [hunk 1]
```diff
@@ -1276,4 +1276,14 @@
 helps['tag'] = """
 type: group
 short-summary: Manage resource tags.
+examples:
+  - name: Creates a tag in the subscription. (autogenerated)
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/resource/_help.py]: autogenerated? Create a tag in the subscription?

Comment (yonzhan) [reply src/azure-cli/azure/cli/command_modules/resource/_help.py]: Why add autogenerated, similar as Aladdin?

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/resource/_help.py]: > Why add autogenerated, similar as Aladdin?

As shown in issue #10867, these examples are provided by Aladdin. Do you think we should remove these flags?

**Comment by Juliehzl** [comment]
@zhoxing-ms @qianwens please help review it.

**Comment by yonzhan** [comment]
add to S171

**Pull request merged**